hypothesis-python/src/hypothesis/_strategies.py

```diff
@@ -1305,6 +1305,25 @@ def from_type(thing):
        other elements in the lookup.
     4. Finally, if ``thing`` has type annotations for all required arguments,
        it is resolved via :func:`~hypothesis.strategies.builds`.
+
+    There is a valuable recipe for leveraging ``from_type`` to generate
```

Comment:
``from_type()``

(doesn't need a cross-reference to the current docstring, does need the standard parentheses)